source/OutfitterPanel.cpp

```diff
@@ -122,7 +148,7 @@ int OutfitterPanel::TileSize() const
 
 int OutfitterPanel::VisibilityCheckboxesSize() const
 {
-	return 30;
+	return 80; // checkboxHeight = 20 x 4 checkboxes = 80;
```

Comment:
In most cases, comments should be separate lines.
```suggestion
	// checkboxHeight = 20 x 4 checkboxes = 80
	return 80;
```